### ISSUE TITLE
I met a problem with the model training

### ISSUE THREAD

**Issue by KindredSpirithub** [linhuixiao/CLIP-VG#8]
After training multiple epochs, I found that the accuracy of the model's test set has always been a very low value. After checking the data side, there was no problem, so I don't know where the problem lies?
![image](https://github.com/linhuixiao/CLIP-VG/assets/38314189/e26e419f-4868-4209-bc1a-2754ecc755c1)

![image](https://github.com/linhuixiao/CLIP-VG/assets/38314189/2a3e69d5-64bf-43a1-95cd-ab4eb5bef19d)

**Comment by linhuixiao** [comment]
Hi, there must be something wrong. Could you replicate the results based on the checkpoint I provided? If it's true,  environmental concerns can be ruled out. In general, after 1 epoch of training, the accuracy can reach 13.0-25.0.

**Comment by KindredSpirithub** [comment]
yes,  i can replicate the results based on the checkpoint you provided, and I didn't make any changes to the source code

**Comment by linhuixiao** [comment]
@KindredSpirithub Hi, I have pulled the code from the repository and tested it in a new environment. The model converged quickly and replicated the results. Thus, the issue you are encountering is probably not a bug in the code; it may be a problem with your environment or configuration. Maybe you can double-check the configuration or environment. 
（I noticed that your training command is not configured exactly according to the readme instructions; you can try using torch.distributed.launch.）

**Comment by KindredSpirithub** [comment]
I just change the  training command by using torch.distributed.launch, then it converge normally. thank you very much
